go.mod

```diff
@@ -1,12 +1,38 @@
 module github.com/omec-project/util_3gpp
 
-go 1.14
+go 1.21
+
+toolchain go1.21.6
```

Comment:
Why is this line required? Is the goal to enable `toolchain` across all repos?

Reply:
I did not realize this was added, I removed it.